bcgov/cas-registration
# 2427 internal users can transfer a facility or operation

[ISSUE 1949](https://github.com/bcgov/cas-registration/issues/1949)
&&
[ISSUE 2427](https://github.com/bcgov/cas-registration/issues/2427)


🗒️ **NOTES:**

- Added `cas_analyst` as the new user role
- Excluded `all` from backend error util if error was a generic backend error (We don't want to see `All:<rest of the error>` in the UI)
- Removed unused jsonSchema
- Fixed `allowedRoles` prop on dashboard tile links
- Updated some fixtures
- We still need to add the cronjob (https://github.com/bcgov/cas-registration/issues/2040) to be able to transfer entities in the future

🧪 **TEST:**

- Log in as cas_analyst
- Click the Transfer link
- Click `Make a Transfer` button
- See transfer form
